roles/stage_items/molecule/default/molecule.yml

```diff
@@ -3,27 +3,21 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
 platforms:
   - name: testme
-    image: centos:7
+    image: docker.io/pycontribs/centos:8
```

Comment:
Something I mentioned offline... should we use the pycontribs/centos:8 container here... or should we be using just the default centos:8

Reply:
I tend to default to using what is officially referenced by the software's documentation or templated configurations unless there's a reason to change it.  I'm open to changing this to use official CentOS images if we can identify a reason to do so.